docs/getting-started/github-apps/snippet--installation.rst

```diff
@@ -0,0 +1,50 @@
+Installing the GitHub App is simple. Navigate to ``/my-tools``, ``/my-workflows``, or ``/my-services`` using the drop down menu in the top right. In these screenshots, we will go via ``/my-tools``, but the process is essentially the same for any of the other options.
+
+.. image:: /assets/images/docs/my-tools.png
+
+Click the ``+`` button on the left hand sidebar.
+
+.. image:: /assets/images/docs/add-tool-button.png
+   :width: 40 %
+
+A window will appear asking how you would like to register your tool, workflow, or service. Select ``Register using GitHub Apps``.
+
+.. image:: /assets/images/docs/register-tool-github-apps.png
+   :width: 50 %
+
+Click ``+ Manage Dockstore Installation on GitHub``. You'll then be redirected to GitHub where you can select which repositories can be accessed by the GitHub app.
+
+.. image:: /assets/images/docs/manage-gh-app-installation.png
+   :width: 50 %
+
+You'll then be redirected to GitHub where you can grant the app access to specific repositories within whatever organization you are installing into. Note that GitHub treats your username as its own "organization." For instance, my GitHub username is aofarrel. If I want to install the GitHub App so it could access aofarrel/mycoolrepo, I would choose the first option here.
+
+.. figure:: /assets/images/docs/gh-app-install-where.png
+   :width: 65 %
+
+   Install our GitHub App on either all repositories in an organization or on specific repositories
+
+After selection an organization, you can select whether to give access to all repositories or only select ones. If the organization you choose is intended to be just for Dockstore tools/workflows/services, or if you know the person who will be adding new entries to Dockstore is not an admin of the organization (see below note on permissions), we recommend choosing all repositories. Otherwise, it is may be more intuitive to select only certain repositories. Click save and you will be taken back to the page you started on in Dockstore -- either  ``/my-tools``, ``/my-workflows``, or ``/my-services``, depending where you started.
```

Comment:
selection of an organization

or

selecting an organization

Reply:
> we recommend choosing all repositories

I don't know about that. I think it would primarily based on content, e.g., if I have one workflow in my org and 20 non-bioinformatics repositories (GUI's, web services, games, etc.), I wouldn't turn it on for all repos.

Reply:
Not selecting all repositories causes a scenario where it is very likely that people will need to keep bothering the admins of the GitHub organization to fix their permissions. I think we should recommend this, but not as strongly as we recommend (say) not making WDL tools. We are still leaving the final decision to our users, but if they do as we recommend then we saved them a potential headache.

Reply:
I don't think we should recommend it, there are too many variables. I'd also prefer that Dockstore not get notifications for a bunch of repos that will never have a .dockstore.yml -- it just puts unnecessary load on our system.

Reply:
That's very true. In that case, we should research exactly how GitHub permissions work so we can guide users (and ourselves) when they do come up. In the meantime I will remove the recommendation.